src/main/java/org/jenkinsci/plugins/parameterizedscheduler/FieldAccessor.java

```diff
@@ -1,29 +0,0 @@
-package org.jenkinsci.plugins.parameterizedscheduler;
-
-import java.lang.reflect.Field;
-
-/**
- * lets work around some final classes I couldn't subclass, but don't want to duplicate everything
- * 
- * @author jameswilson
- *
- */
-public class FieldAccessor {
```

Comment:
Formally it breaks Binary compatibility.
There is no usages outside the plugin, so it should be fine in this case